lib/puppet/type/postgresql_psql.rb

```diff
@@ -135,11 +135,7 @@ def matches(value)
   end
 
   autorequire(:anchor) do
```

Comment:
as an alternative, can we maybe autorequire the postgresql_conn_validator resource?

Reply:
That quite make sense.  I'll dig into this.